docs/USAGE-GUIDE.md

```diff
@@ -370,6 +370,14 @@ s2n-tls provides multiple different methods to get the TLS protocol version of t
 * `s2n_connection_get_server_protocol_version()`: The highest TLS protocol version the server supports.
 * `s2n_connection_get_client_protocol_version()`: The highest TLS protocol version the client advertised.
 
+## Config
+
+`s2n_config` objects are used to change the default settings of a s2n-tls connection. Use `s2n_config_new()` to create a new config object and `s2n_config_free()` to free the object when no longer needed. It is not necessary to create a config object per connection; one config object can be used for many connections. To associate a connection with a config call `s2n_connection_set_config()`. Most commonly, a `s2n_config` object is used to set the certificate key pair for authentication and change the default security policy. See the sections for [certificates](#certificates-and-authentication) and [security policies](#security-policies) for more information on those settings.
```

Comment:
This might not be important, but I wonder if the language for using configs on more than one connection should be more forceful, to better distinguish between the connection configuration APIs? Like, "It is not necessary to create a config object per connection; one config object _should_ be used for many connections. See Overriding the Config to configure behavior for a single connection" or something. Not sure.

Reply:
Re-worded this paragraph to be more forceful.